eos-s3/btn_counter/README.rst

```diff
@@ -7,4 +7,5 @@ counter example, run the following command:
 .. code-block:: bash
    :name: eos-s3-counter
 
+   export FPGA_FAM=eos-s3
```

Comment:
This is required only to fix CI. For some reason the `FPGA_FAM` env variable is dropped when `make` is executed.
It is visible in [this](https://github.com/antmicro/f4pga-examples/runs/6831396204?check_suite_focus=true#step:9:36) CI run where echos in docs `docs/building-examples.rst` and `eos-s3/btn_counter/README.rst` show existing and valid `FPGA_FAM` while the echos in the makefile don't show this variable and that was forcing f4pga to use default wrappers that are meant for xc7.

Reply:
This might need further investigation... See https://github.com/chipsalliance/f4pga-examples/blob/main/.github/scripts/build-examples.sh#L88. `snippets` includes `activate_env`, which has `fpga-fam-$fpga_family`.
Can you please add a FIXME comment in L9, to ensure we don't forget about it?

Reply:
Added the comment